CHANGELOG.unreleased.md

```diff
@@ -28,6 +28,15 @@ Section headings should be at level 3 (e.g. `### Added`).
 - The server now supports fetching artifact files by providing additional collection information; updated the artifacts api to use the new endpoints instead (@ibindlish in https://github.com/wandb/wandb/pull/9551)
 - Paginated methods (and underlying paginators) that accept a `per_page` argument now only accept `int` values.  Default `per_page` values are set directly in method signatures, and explicitly passing `None` is no longer supported (@tonyyli-wandb in https://github.com/wandb/wandb/pull/9201)
 
+### Deprecated
+
+- The following `wandb.Run` methods are deprecated in favor of properties and will be removed in a future release (@kptkin in https://github.com/wandb/wandb/pull/8925):
+    - `run.project_name()` is deprecated use instead `run.project`
+    - `run.get_url()` method is deprecated use instead `run.url`
+    - `run.get_project_url()` method is deprecated use instead `run.project_url`
+    - `run.get_sweep_url()` method is deprecated use instead `run.sweep_url`
```

Comment:
The wording of "X is deprecated use instead Y" is awkward. Replace "use instead" by "in favor of".